server/config/passport.js

```diff
@@ -15,7 +15,8 @@ passport.use(new LocalStrategy({
     console.log(username)
     return UserDB
       .findOne({
-        where: { username: username }
+        where: { username: username },
+        raw: true
```

Comment:
This sets Sequelize to load the user as a plain object and not an instance of a class.